readme.md

```diff
@@ -54,6 +54,16 @@
 | ------: | -------: | -----: |
 |   1 - 8 |       13 |    90m |
 
+### [Dixit](https://en.wikipedia.org/wiki/Dixit_(card_game))
+
+> Using a deck of cards illustrated with dreamlike images, players select cards that match a title suggested by the "storyteller", and attempt to guess which card the "storyteller" selected.
+
+![Dixit_game_image](https://upload.wikimedia.org/wikipedia/en/thumb/7/7b/Dixitgame.jpg/200px-Dixitgame.jpg)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|   3 - 8 |       8 |    30m |
```

Comment:
As you can see from [the Travis CI build](https://travis-ci.org/edm00se/awesome-board-games/builds/446674990#L486-L487), which also runs some format linting, this line could be massaged for consistent formatting of the table. Adding a single space in the min age column should take care of it.

```suggestion
|   3 - 8 |        8 |    30m |
```